### ISSUE TITLE
Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by lukeapage** [wix/eslint-plugin-lodash#118]
Sorry I don't have a reduced test case, but I do have a stack trace, maybe thats enough? version is 2.1.6

```
TypeError: Cannot read property 'name' of undefined
  at isOnlyParamInvocationsWithOperator (c:\path\to\node_modules\eslint-plugin-lodash\lib\rules\prefer-over-quantifier.js:50:71)
  at isOnlyParamInvocationsWithOperator (c:\path\to\node_modules\eslint-plugin-lodash\lib\rules\prefer-over-quantifier.js:53:54)
  at reportIfConnectiveOfParamInvocations (c:\path\to\node_modules\eslint-plugin-lodash\lib\rules\prefer-over-quantifier.js:67:21)
  at c:\path\to\node_modules\eslint-plugin-lodash\lib\rules\prefer-over-quantifier.js:85:17
  at EventEmitter.<anonymous> (c:\path\to\node_modules\eslint-plugin-lodash\lib\util\lodashUtil.js:92:13)
  at emitOne (events.js:95:20)
  at EventEmitter.emit (events.js:182:7)
  at NodeEventGenerator.enterNode (c:\path\to\node_modules\eslint\lib\util\node-event-generator.js:40:22)
  at CodePathAnalyzer.enterNode (c:\path\to\node_modules\eslint\lib\code-path-analysis\code-path-analyzer.js:608:23)
  at CommentEventGenerator.enterNode (c:\path\to\node_modules\eslint\lib\util\comment-event-generator.js:97:23)
  at Controller.enter (c:\path\to\node_modules\eslint\lib\eslint.js:898:36)
  at Controller.__execute (c:\path\to\node_modules\eslint\node_modules\estraverse\estraverse.js:397:31)
  at Controller.traverse (c:\path\to\node_modules\eslint\node_modules\estraverse\estraverse.js:501:28)
  at Controller.Traverser.controller.traverse (c:\path\to\node_modules\eslint\lib\util\traverser.js:36:33)
  at EventEmitter.module.exports.api.verify (c:\path\to\node_modules\eslint\lib\eslint.js:895:23)
  at processText (c:\path\to\node_modules\eslint\lib\cli-engine.js:264:31)
  at processFile (c:\path\to\node_modules\eslint\lib\cli-engine.js:303:18)
  at executeOnFile (c:\path\to\node_modules\eslint\lib\cli-engine.js:674:25)
  at c:\path\to\node_modules\eslint\lib\cli-engine.js:712:13
  at Array.forEach (native)
  at CLIEngine.executeOnFiles (c:\path\to\node_modules\eslint\lib\cli-engine.js:711:18)
  at Object.<anonymous> (c:\path\to\node_modules\grunt-eslint\tasks\eslint.js:29:20)
  at Object.<anonymous> (c:\path\to\node_modules\grunt\lib\grunt\task.js:255:15)
  at Object.thisTask.fn (c:\path\to\node_modules\grunt\lib\grunt\task.js:73:16)
  at Object.<anonymous> (c:\path\to\node_modules\grunt\lib\util\task.js:294:30)
  at Task.runTaskFn (c:\path\to\node_modules\grunt\lib\util\task.js:244:24)
  at Task.<anonymous> (c:\path\to\node_modules\grunt\lib\util\task.js:293:12)
  at Task.start (c:\path\to\node_modules\grunt\lib\util\task.js:302:5)
  at Object.grunt.tasks (c:\path\to\node_modules\grunt\lib\grunt.js:155:8)
  at Object.module.exports [as cli] (c:\path\to\node_modules\grunt\lib\grunt\cli.js:27:9)
  at Object.<anonymous> (C:\npmprefix\node_modules\grunt-cli\bin\grunt:44:20)
  at Module._compile (module.js:413:34)
  at Object.Module._extensions..js (module.js:422:10)
  at Module.load (module.js:357:32)
  at Function.Module._load (module.js:314:12)
  at Function.Module.runMain (module.js:447:10)
  at startup (node.js:146:18)
  at node.js:404:3

```

**Comment by lukeapage** [comment]
I'll try and add more logging and get info later on.

**Comment by lukeapage** [comment]
the code causing this is

```
...
    firstPredefinedList() {
        return _.find(this.all(), function(ab) {
            return ab.isPredefined() && ab.isCommonTypeEnabled();
        });
    },
...
```

**Comment by ganimomer** [comment]
Thanks for the catch! Fixed in `2.1.7`.
